pkg/service/keystore/model.go

```diff
@@ -35,3 +35,10 @@ type GetKeyDetailsResponse struct {
 	Controller string
 	CreatedAt  string
 }
+
+type DeleteKeyRequest struct {
+	ID string
+}
+
+type DeleteKeyResponse struct {
```

Comment:
is this necessary?

Reply:
I've updated the following to just send an error back instead of a response.